Add a new hook

openrichardfb opened the pull request:
Differential Revision: D45220984

albanD reviewed (approved):
The aten change looks good.
I assume you got a review from the profiler side already?

openrichardfb committed:
Add a new hook (#99854)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/99854

Reviewed By: aaronenyeshi

Differential Revision: D45220984

fbshipit-source-id: 42442bd7b8e961d2a0f7b03a04a6c7ce2b2fb3cc

openrichardfb commented:
> The aten change looks good. I assume you got a review from the profiler side already?

Thanks Alban! I have got a review accepted internally from the Kineto side (Aaron).

openrichardfb commented:
@pytorchmergebot merge

openrichardfb commented:
@pytorchbot label "topic: not user facing"

openrichardfb commented:
@pytorchmergebot merge
